orc8r/cloud/go/obsidian/obsidian/main.go

```diff
@@ -53,6 +53,10 @@ func main() {
 		&obsidian.StaticFolder, "static_folder", obsidian.DefaultStaticFolder,
 		"Folder containing the static files served",
 	)
+	flag.BoolVar(
```

Comment:
Sorry maybe I was unclear before: we want the config values to come from the obisidian.yml config file, and here (in main.go) we set the relevant config.go global based on the value from the config file. No CLI arguments involved

As an aside, as we previously discussed, the way we handle configs in orc8r is a bit haphazard, here in particular. But let's go with the way I describe. This will allow operators to flip things by editing their config file, rather than having to potentially upload new Helm charts (idk if it would be easy or hard, but I know config changes are intentionally easy)